package.yaml

```diff
@@ -24,28 +24,42 @@ dependencies:
 - base >= 4.7 && < 5
 - directory
 - esqueleto
+- hspec
 - monad-logger
 - persistent
 - persistent-postgresql
 - persistent-qq
 - persistent-template
+- strive
 - template-haskell
 - text
 - time
 - tmp-postgres
-- hspec
 
 default-extensions:
+- DerivingStrategies
 - GADTs
 - GeneralizedNewtypeDeriving
 - MultiParamTypeClasses
 - NoImplicitPrelude
 - OverloadedStrings
 - QuasiQuotes
+- StandaloneDeriving
```

Comment:
These two extensions are required by the `persistent-template-2.8.0` update. They're also just generally good, so they're fine to stay in.